### ISSUE TITLE
Schema.org A11Y reference in Webography 

### ISSUE THREAD

**Issue by LaZay** [w3c/a11y-discov-vocab#62]
The [A.1 Informative references](https://w3c.github.io/a11y-discov-vocab/crosswalk/#informative-references) section is lacking the link to [Schema.org Accessibility Properties for Discoverability Vocabulary](https://www.w3.org/2021/a11y-discov-vocab/latest/).

It is highly important to correctly source this standard, as [Schema.org ](https://schema.org/) web site does not provide the list of ALL accessibility **properties** it has defined. Many people think there is one property = "`accessibilityFeature`", which is not the case. There are 6 accessiblity properties in Schema:
1. [accessMode](https://schema.org/accessMode)
2. [accessModeSufficient](https://schema.org/accessModeSufficient)
3. [accessibilityControl](https://schema.org/accessibilityControl)
4. [accessibilityHazard](https://schema.org/accessibilityHazard)
5. [accessibilityFeature](https://schema.org/accessibilityFeature)
6. [accessibilityAPI](https://schema.org/accessibilityAPI)

With N values for EACH. Futhermore, authorized **values** for each property have changed location in the pages. They are no more directly provided for each property's page. They are hidden behind this sentence now: _"Values should be drawn from the [approved vocabulary](https://www.w3.org/2021/a11y-discov-vocab/latest/#accessibilityAPI-vocabulary)"_

Only the [Schema.org Accessibility Properties for Discoverability Vocabulary](https://www.w3.org/2021/a11y-discov-vocab/latest/) provides the full picture about the way Schema handles accessibility metadata (i.e. properties, and values).

**Comment by clapierre** [comment]
We believe this has been recently addressed please reopen with additional information if this is still and outstanding issue.
